src/FieldEncryptProcessEntities.php

```diff
@@ -101,26 +135,72 @@ public function decryptEntity(ContentEntityInterface $entity) {
    * @return string
    *   The processed value.
    */
-  protected function processValue($value = '', EncryptionProfileInterface $encryption_profile, $op = 'encrypt') {
+  protected function processValue(ContentEntityInterface $entity, FieldItemListInterface $field, $property_name, $value = '', EncryptionProfileInterface $encryption_profile, $op = 'encrypt') {
     // Do not modify empty strings.
     if ($value === ''){
       return '';
     }
 
     if ($op === 'encrypt') {
-      return base64_encode($this->encryptService->encrypt($value, $encryption_profile));
+      // Encrypt property value.
+      $processed_value = base64_encode($this->encryptService->encrypt($value, $encryption_profile));
+      // Save encrypted value in EncryptedFieldValue entity.
+      $this->encryptedFieldValueManager->saveEncryptedFieldValue($entity, $field->getName(), $property_name, $processed_value);
+      // Return value to store for unencrypted property.
+      // We can't set this to NULL, because then the field values are not saved,
+      // so we can replace them with their unencrypted value on load.
+      return '[ENCRYPTED]';
```

Comment:
This threw me for a loop at first but I totally understand this based on your comments. Thank you for including it